src/Asset/Button/Button.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styles from '../CSS/Commonmodule.css';
+// import Images from '../Images/Images';
+
+const colors = ['#0088FE', '#00C49F', '#FFBB28'];
```

Comment:
Create separate constants folder at src level and create file called COLORS.js within it and keep all common color code exported as object which are going to reuse as color code